datahub-web-react/vite.config.ts

```diff
@@ -5,6 +5,28 @@ import svgr from 'vite-plugin-svgr';
 import macrosPlugin from 'vite-plugin-babel-macros';
 import { viteStaticCopy } from 'vite-plugin-static-copy';
 
+const injectMeticulous = () => {
+    if (!process.env.REACT_APP_METICULOUS_PROJECT_TOKEN) {
+        return null;
+    }
```

Comment:
by default, any user developing locally will not have meticulous (a frontend regression testing tool) recording for their security reasons